### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by Brandonbr1**
add PULL_REQUEST_TEMPLATE.md because it is good community standards. 😃

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## Description
+
+- Please include a summary of the change and which issue is fixed.
+
+# additional info
```
Comment (dada513) [inline .github/PULL_REQUEST_TEMPLATE.md:5]:
```suggestion
# Additional info
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+
+
```
Comment (dada513) [inline .github/PULL_REQUEST_TEMPLATE.md:21]:
```suggestion

```
I think this is obvious from the description of the PR and labels added, so seems unnecessary work for the contributor

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
```
Comment (dada513) [inline .github/PULL_REQUEST_TEMPLATE.md:29]:
```suggestion

```
There's no automatic tests for this extension rn so it's not needed

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+- [ ] New and existing unit tests pass locally with my changes
```
Comment (dada513) [inline .github/PULL_REQUEST_TEMPLATE.md:30]:
```suggestion

```
Same as above

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+- [ ] Any dependent changes have been merged and published in downstream modules
+- [ ] I have checked my code and corrected any misspellings
+
+# Did you test it on any of the browsers?
```
Comment (dada513) [inline .github/PULL_REQUEST_TEMPLATE.md:34]:
```suggestion
# What browsers did you test it on?
```

**Review by dada513 — CHANGES_REQUESTED** [review]

**Comment by Brandonbr1** [comment]
@dada513  changes has been done😃

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+## Description
+
+- Please include a summary of the change and which issue is fixed.
+
+# Additional info
+- Please also include relevant  context.
```
Comment (dada513) [inline .github/PULL_REQUEST_TEMPLATE.md:6]:
```suggestion
- Please also include relevant context.
```

**Review by dada513 — CHANGES_REQUESTED** [review]

**Pull request merged**